tekton/tasks/binary-container-cachi2.yaml

```diff
@@ -113,7 +114,11 @@ spec:
           # we can rely on presence of this file
           SBOMS+=("${REMOTE_SOURCE_PATH}/bom.json")
 
-          rm -fr app/.git/   # remove git directory
+          if [ -f "${CACHI2_INCLUDE_GIT_DIR_FILE}" ]; then
```

Comment:
something here is wrong, file exists, but this condition wasn't evaluated as true

however it is true in my local testing, magic

Reply:
because I'm stupid that's why, I haven't updated reference to this task for testing PR

Reply:
it's ready, it's was harder to tests it than write it